build/deps/gen/deps/dep_capnp_cpp.MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+# WARNING: THIS FILE IS AUTOGENERATED BY update-deps.py DO NOT EDIT
+
+http = use_extension("@//:build/exts/http.bzl", "http")
+http.archive(
```

Comment:
Using this extension changes the canonical name of the repo to add `+http+` in there. Is this going to cause us any trouble?